index.js

```diff
@@ -34,6 +34,7 @@ ffmpegPlatform.prototype.configureAccessory = function(accessory) {
 ffmpegPlatform.prototype.didFinishLaunching = function() {
   var self = this;
   var videoProcessor = self.config.videoProcessor || 'ffmpeg';
+  var interfaceName = self.config.interfaceName;
```

Comment:
Can you explicitly set a fallback value here since this is optional?

Reply:
Yes, any value that evaluates to false should work, e.g. an empty string.

```suggestion
  var interfaceName = self.config.interfaceName || '';
```

Reply:
Looks good. Can you apply the change to this PR and update the PR? Thanks 🙏